### ISSUE TITLE
Possible XLN/DRL conversion error 

### ISSUE THREAD

**Issue by tecchese** [sn4k3/UVtools#676]
### System

```Text
UVtools v3.12.0 X64
Operative system: Microsoft Windows 10.0.19045 X64
Processor: Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
Processor cores: 8
Memory RAM: 5.60 / 15.89 GB
Runtime: win10-x64
Framework: .NET 6.0.14
AvaloniaUI: 0.10.18
OpenCV: 4.6.0

Path:       C:\Program Files\UVtools\
Executable: C:\Program Files\UVtools\UVtools.exe
Loaded file: db9_conn_copy.pwma [Version: 516] [Class: PhotonWorkshopFile]
```

### Description of the bug

PCB Excellon file is not converted (blank image) in PCB exposure tool.

see:
![drl](https://user-images.githubusercontent.com/107458958/226097537-8ef6128b-ce3d-4ddc-98d9-edd819a96f2e.JPG)

GBR is ok:

![gbl](https://user-images.githubusercontent.com/107458958/226097590-f2c0e657-d618-43dc-a28b-448c42b4157d.JPG)

Saved excellon and GBR bottom layout, then saved as PDF and converted again in GBR with Gerbview: ok...

![pdf](https://user-images.githubusercontent.com/107458958/226097649-e579feaf-08dd-4a8a-9818-5bcf0788f01c.JPG)

### How to reproduce

Please try by yourself, I attached ZIP files
Thank you for your support.

### Files

[test.zip](https://github.com/sn4k3/UVtools/files/11008118/test.zip)
[untitled_2023-03-12.zip](https://github.com/sn4k3/UVtools/files/11008119/untitled_2023-03-12.zip)

**Comment by sn4k3** [comment]
Problem is on drill file, it lacks of G05 == turn on drill mode and if drill is off it should not drill at all. If you insert G05 after M71 it will work.

I can latter ignore the requirement of G05 to do the holes

**Comment by tecchese** [comment]
Yes, if I insert manually G05 after M71 it works but there are still problems (see video).
[Untitled.zip](https://github.com/sn4k3/UVtools/files/11010314/Untitled.zip)
The method of subtracting the holes by inverting the color for the imported Excellon file is confusing and the preview doesn't help. Also you need to add the XLN extension (used by EagleCAD) for importing Excellon files.

**Comment by sn4k3** [comment]
> it works but there are still problems (see video).

What is the problem exactly? I run your drl + grl and I got the correct output as your first image of converted pdf to gbr.

> Also you need to add the XLN extension (used by EagleCAD) for importing Excellon files.

I will take a look

**Comment by sn4k3** [comment]
> It might be wiser to select which layer can be inverted individually, not for all.

You can, there is a check box for inverting individual gerbers, does that work for you?

![image](https://user-images.githubusercontent.com/113281/226652637-f24812b1-c130-442d-be5e-74faa897cb2a.png)

**Comment by sn4k3** [comment]
Result of no inverting:

![image](https://user-images.githubusercontent.com/113281/226654413-b3272ef2-9ddb-4913-928e-add6b68c2865.png)

Result of inverting only one gerber:
![image](https://user-images.githubusercontent.com/113281/226654468-1e148dbb-75b8-454d-af16-65013500d6a0.png)
![image](https://user-images.githubusercontent.com/113281/226654485-040dcdd6-14bd-489a-9b2a-2ba9acb3df99.png)

**Comment by tecchese** [comment]
For me it does not work...
I get no result when I click on the checkbox indicated by the red arrow...
this is the question...
In addition if I select a layer the size scale data disappears
![uvtest](https://user-images.githubusercontent.com/107458958/226688855-147f7120-f250-43fc-92e7-418f0c3485d5.JPG)
Also on the top you can notice the description partially cut.

**Comment by sn4k3** [comment]
For the individual invert to work, order does matter as it only inverts drawings instead of whole layer, that means where white black will be drawn, so order whites in first place is important.

> In addition if I select a layer the size scale data disappears

Looks like a UI problem, you are using the normal theme right? Can you set back to the Fluent theme?

**Comment by tecchese** [comment]
When I changed theme I get this:
![crashra](https://user-images.githubusercontent.com/107458958/226691658-6c509f27-a52a-4412-930a-d5e28fb68369.JPG)

**Comment by sn4k3** [comment]
Yes you need to restart UVtools. If unable to change and save you need to go to settings file and change there or remove the theme key

**Comment by tecchese** [comment]
Ok, I selected the default Fluent Theme and everything seems to work...but it isn't true unfortunately...
see this pic:
![pvbuvtools](https://user-images.githubusercontent.com/107458958/226843945-a0b2923f-4761-4222-9278-4ddde2d11d58.JPG)

All the invert level checkboxes (except drill level) on the right always stay active when I checked them repeatedly and there is no way to go back (i.e. to uncheck them)... very strange behaviour.

Thanks for the awesome support and patience.
